docs/superpowers/plans/2026-08-15-gtid-aurora-admin-coverage.md

```
@@ -0,0 +1,268 @@
+
+- Modify: test/tap/groups/groups.json line 420
+- Test: test/tap/tests/test_gtid_from_ok-t.cpp
+
+Interfaces:
+
+- Consumes: the existing causal min_gtid SELECT workload and MySQL 8.4 group.
+- Produces: GCOV execution of the existing TAP in mysql84-g5.
+
+- [ ] Step 1: Build the unchanged GTID TAP with GCOV
+
+~~~
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)